### ISSUE TITLE
v8.0.0-RC1: Image block sits outside of perceived conatiner in editor (Twenty Twenty theme)

### ISSUE THREAD

**Issue by andrewserong** [WordPress/gutenberg#21961]
**Describe the bug**
In v8.0.0-RC1 and the Twenty Twenty theme enabled, within the editor the image block sits outside of the central 'container' of the page.

This issue is related to #21885 and #21911 (but this issue is about when no alignment is set).

It looks like this is to do with the change in https://github.com/WordPress/gutenberg/pull/21822 where the wrapper `div` was removed from the Image block in the editor. This change means that the [following styling in the Twenty Twenty theme](https://core.trac.wordpress.org/browser/trunk/src/wp-content/themes/twentytwenty/assets/css/editor-style-block.css#L366) now overrides the margins that should centre the block:

```css
.editor-styles-wrapper figure {
  margin: 0;
}
```

Whereas if you disable the plugin, then the wrapper div element is targeted by 

```
.edit-post-visual-editor .block-editor-block-list__block {
```

This particular selector was removed in https://github.com/WordPress/gutenberg/pull/20951 at [this line](https://github.com/WordPress/gutenberg/pull/20951/files#diff-947e2eca7278f9a543620b68d7187d09L40). A solution to this issue could be either to fix this in Twenty Twenty, or add back in the div wrapper to the Image block to mitigate the specificity issue.

**To reproduce**
1. Activate the Twenty Twenty theme on your site
2. Add an Image block and select any image and set it to left aligned
3. Instead of the image block being centered within the perceived container of the page, the image is left aligned outside of that 'container'.

**Expected behavior**
The image block to sit within the perceived 'container' of the page in Twenty Twenty as it does on the front end.

**Screenshots**

### Before (WordPress 5.4, plugin disabled)

![image](https://user-images.githubusercontent.com/14988353/80554224-cad01b00-8a0f-11ea-92ef-cf918f8a264d.png)

### After (WordPress 5.4, plugin v8.0.0-RC1 enabled)

![image](https://user-images.githubusercontent.com/14988353/80554182-a3794e00-8a0f-11ea-9a5b-44766e82b967.png)

**Editor version (please complete the following information):**
- WordPress version: 5.4, Gutenberg v8.0.0-RC1

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome 81

**Comment by andrewserong** [comment]
@aduth — I'm not sure if this is a blocking issue, but thought I'd mention it since you released RC 1 🙂

**Comment by aduth** [comment]
@youknowriad As author of #21822 and #20951, do you have any thoughts on what might need to change here?

**Comment by aduth** [comment]
Possibly related fix: #21971

**Comment by nosolosw** [comment]
Tested and can confirm that https://github.com/WordPress/gutenberg/pull/21971 fixes this as well.

**Comment by andrewserong** [comment]
Thank you for the quick fix on this one and getting it in for v8.0.0! 🎉
